src/main/java/uk/gov/hmcts/ccd/domain/model/definition/SearchResultField.java

```diff
@@ -53,4 +54,13 @@ public boolean isMetadata() {
     public void setMetadata(boolean metadata) {
         this.metadata = metadata;
     }
+
+    @JsonProperty("role")
```

Comment:
can we add this annotation to the field to be consistent with the rest of the properties?

Reply:
done